config/v1/types.go

```diff
@@ -310,3 +310,74 @@ type DelegatedAuthorization struct {
 	// disabled indicates that authorization should be disabled.  By default it will use delegated authorization.
 	Disabled bool `json:"disabled,omitempty"`
 }
+type RequiredHSTSPolicy struct {
+	// namespaceSelector specifies a label selector such that the policy applies only to those objects that
```

Comment:
```suggestion
	// namespaceSelector specifies a label selector such that the policy applies only to those routes that
```
This just applies to route resources, right?